services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceHelixAdmin.java

```diff
@@ -2621,7 +2618,7 @@ private Pair<Boolean, Version> addVersion(
             }
 
             Properties veniceViewProperties = new Properties();
-            veniceViewProperties.put(SUB_PARTITION_COUNT, subPartitionCount);
+            veniceViewProperties.put(SUB_PARTITION_COUNT, numberOfPartitions);
```

Comment:
maybe rename SUB_PARTITION_COUNT too?

Reply:
I don't know if View is still lingering in our system; So I don't want to rename the config key directly. But I can rename the variable to PARTITION_COUNT.

Reply:
Essentially, I think we can delete them altogether if we are in the near term determined that View is not going to be useful.